pkg/services/auth/idimpl/service.go

```diff
@@ -42,9 +42,7 @@ func ProvideService(
 		nsMapper: request.GetNamespaceMapper(cfg),
 	}
 
-	if features.IsEnabledGlobally(featuremgmt.FlagIdForwarding) {
```

Comment:
Nit: Looks like `features` isn't needed anymore so you could drop it as a dependency